cmd/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"os"
+	"os/signal"
+	"syscall"
+
+	task "github.com/Extrarius/tido"
+	"github.com/Extrarius/tido/pkg/handler"
+	"github.com/Extrarius/tido/pkg/repository"
+	"github.com/Extrarius/tido/pkg/service"
+
+	"github.com/joho/godotenv"
+	"github.com/sirupsen/logrus"
+	_ "modernc.org/sqlite"
+)
+
+func main() {
+	// Устанавливает формат логов в JSON
+	logrus.SetFormatter(new(logrus.JSONFormatter))
+
+	if err := godotenv.Load(); err != nil {
+		logrus.Fatalf("ошибка при загрузке переменных: %s", err.Error())
+	}
+
+	// Устанавливаем флаг для порта при запуске. Например: go run ./cmd -port="7540"
+	port := flag.String("port", os.Getenv("TODO_PORT"), "")
+	flag.Parse()
+
+	db, err := repository.NewDB()
+	if err != nil {
+		logrus.Fatalf("не удалось инициализировать базу данных: %s", err.Error())
+	}
+
+	// Устанавливаем зависимости
+	repos := repository.NewRepository(db)
+	services := service.NewService(repos)
+	handlers := handler.NewHandler(services)
+
+	srv := new(task.Server)

```

Comment:
Обычно делают функцию `New`, которая возвращает созданную структуру, а `Run` уже чисто для запуска. 
```suggestion
	srv := task.New(*port, handlers.InitRoutes())
```

Тогда внутри будет

```golang
func New(port string, handler http.Handler) Server {
	return Server{&http.Server{
		Addr:    ":" + port,
		Handler: handler,
	}}
}
func (s *Server) Run() error {
	return s.httpServer.ListenAndServe()
}
```